www/js/brushes/brushes-controllers.js

```diff
@@ -1,6 +1,7 @@
 angular.module('app')
 
-.controller('BrushesCtrl', function($scope, BrushesServices) {
+.controller('BrushesCtrl', function($scope, BrushesServices, SoundServices,
+            $ionicPlatform, $timeout) {
   var vm = $scope;
   var colors = ["#919191", "#8F00FF", "#00CFFF", "#F900F3", "#FC0000" , "#FFA300", "#0600FF", "#00C10E" , "#FFF203", "#000000"];
```

Comment:
should be stored in constant where we can modify later without touching codebase